README.md

```diff
@@ -1,70 +1,9 @@
-FAC finds freelance work for its graduates and takes a proportion as a fee, as any agency would. The amount is decided on a project-by-project basis. You do not need to give a proportion of your earnings to FAC if you have moved away and are finding work for yourself, or if you are earning less than the London Living Wage.   
-
-**If I employ another FAC graduate to work with me, do I have to give a proportion of their earnings to FAC?**    
-Like any other employer, you will need to have a recruitment agreement with FAC (although the terms are likely to be more favourable than for other companies).
-
-**Can I run my company from within the FAC workspace and do I have to give a proportion of the earnings to FAC?**   
-If you wish to run a company that employs non-FAC staff, then you will need to get agreement from the FAC directors, who must be satisfied that the arrangement is of benefit to FAC. 
-
-**If I am employed elsewhere, does that mean I'm no longer a part of FAC?**    
-Not at all. As long as you are prepared to help out, you will always continue to be welcome. Wherever you are, we always want you to help with FAC programmes or to set up events, meetup groups and study programmes under the FAC banner (and jointly with your own company or employer, if you like).
-
-**I am moving abroad. Do I still need to refer prospective employers to FAC?**    
-Not unless we have a presence in the city where you work, but do please set up events, meetup groups and study programmes under the FAC banner and keep us posted.
-
-**Will FAC help me find work quickly?**    
-FAC will try to support you in your job hunt and mentor you through interviews, but we do not currently have the resources to hire a dedicated recruiter. As we build up a network of partner companies and agency relationships, active recruitment for recent graduates may become less of an issue, but in the short term graduates may have less support than we would like.
-
-**If I tell FAC about a job I want to apply for, will FAC tell other graduates about the job and increase competition for it?**  
-We think the fairest thing would be to only advertise this job after a suitable period of time has passed. 
-
-**This rule makes me uneasy. I don't want to involve FAC in my conversations with prospective employers.**    
-We understand this, but as our primary source of income, this is currently a condition of joining the programme.
-
-**Are your rules compulsory for all students?**    
-We encourage current students to abide by them, but it is now a condition of accepting an offer to study at FAC.
+In addition, each place on the Founders & Coders programme costs us about £2,500 to provide. We ask you to consider either seeking employment with one of our partners or, alternatively, making a regular voluntary contribution after you graduate as a way of *'paying it forward'*, so that we can provide a place for someone in a future cohort.
```

Comment:
Looks like you've already made the change we'd discussed here and honestly, I :heart: how you've put it. Couldn't have said it better myself.

Just the words *paying it forward* are so indicative of our ethos, they have me like 😍